doc/api/fs.md

```diff
@@ -182,7 +182,8 @@ added: v0.1.93
 
 * `fd` {integer} Integer file descriptor used by the ReadStream.
 
-Emitted when the ReadStream's file is opened.
+Emitted when the `ReadStream`'s underlying file descriptor has been closed
+using the `fs.close()` method.
```

Comment:
I think this last part can/should be omitted. Just end on 'has been closed.'

Reply:
I fully agree but I copied that markdown verbatim from the `v4.x` and `v8.x` docs. As such, I’d either suggest changing it here but also in those spots as well, or else leaving it intact here.

Reply:
But yeah, that last bit seems unnecessary and, honestly, a bit awkward.

Reply:
How do is the upstreaming of commits to other version branches typically handled for this repo?  For example, if I changed it here in `v6.x`, would I submit separate PRs for all of the `v7.x`, `v8.x`, and `v9.x` branches, or do changes to some of them get upstream/merged into others?

Reply:
Make the change in master and it will be backported at some point.

Reply:
@mscdex How far do you backport? Should I rebase this PR to be targeting `master` instead of directly to `6.x`, or this PR **plus** another PR targeting `master`?

Reply:
Well this particular change was made in other branches inadvertently(?) in https://github.com/nodejs/node/pull/11331 it seems. So this PR is fine for now for consistency.

If you want to submit another PR to remove the last portion of this sentence, you'd file a PR against master and it will automatically trickle down to the versioned branches by releasers.

Reply:
Sounds good. :+1:

Reply:
Followup work submitted as PR #15800.